docs/source/dev/tracing/tracing_index.rst

```diff
@@ -0,0 +1,23 @@
+Tracing vLLM
```

Comment:
I would suggest naming this Function Tracing so it won't be confused with the OpenTelemetry tracing for metrics

Reply:
Thanks for the review.